app/gui2/src/components/GraphEditor/GraphVisualization.vue

```diff
@@ -36,6 +39,62 @@ const emit = defineEmits<{
   setVisualizationVisible: [visible: boolean]
 }>()
 
+const visPreprocessor = ref(DEFAULT_VISUALIZATION_CONFIGURATION)
+const error = ref<Error>()
+
+const projectStore = useProjectStore()
+const graphStore = useGraphStore()
+const visualizationStore = useVisualizationStore()
+
+const expressionInfo = computed(
+  () => props.expressionId && graphStore.db.getExpressionInfo(props.expressionId),
+)
+const typeName = computed(() => expressionInfo.value?.typename ?? 'Any')
+
+/** The Visualization ID for the method to get the default visualization. */
+const defaultVisualizationVisualizationId = random.uuidv4() as Uuid
```

Comment:
Why double `Visualization`? I think single is enough (it's obvious it's visualization ID... at least for me).

Reply:
my thought process was, this is the visualization id to get the default visualization - not some kind of fallback visualization id for the *actual* visualization... but yeah it is kinda long